examples/hybrid/sphere/limiters_deformation_flow.jl

```diff
@@ -0,0 +1,621 @@
+    R = 6.37122e6,
+    zlim = (0, 12.0e3);
+    helem = 4,
+    zelem = 12,
+    npoly = 4,
+)
+    FT = Float64
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(zlim[1]),
+        Geometry.ZPoint{FT}(zlim[2]);
+        boundary_names = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, nelems = zelem)
+    vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+
+    horzdomain = Domains.SphereDomain(R)
+    horzmesh = Meshes.EquiangularCubedSphere(horzdomain, helem)
+    horztopology = Topologies.Topology2D(horzmesh)
+    quad = Spaces.Quadratures.GLL{npoly + 1}()
+    horzspace = Spaces.SpectralElementSpace2D(horztopology, quad)
+
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+    return (horzspace, hv_center_space, hv_face_space)
+end
+
+# set up 3D domain
+horzspace, hv_center_space, hv_face_space =
+    sphere_3D(helem = helems, zelem = zelems)
+global_geom = horzspace.global_geometry
+topology = horzspace.topology
+
+# Extract coordinates
+center_coords = Fields.coordinate_field(hv_center_space)
+face_coords = Fields.coordinate_field(hv_face_space)
+
+# Initialize variables needed for limiters
+horz_n_elems = Topologies.nlocalelems(horzspace.topology)
+min_q1 = zeros(horz_n_elems, zelems)
+max_q1 = zeros(horz_n_elems, zelems)
+min_q2 = zeros(horz_n_elems, zelems)
+max_q2 = zeros(horz_n_elems, zelems)
+min_q3 = zeros(horz_n_elems, zelems)
+max_q3 = zeros(horz_n_elems, zelems)
+min_q4 = zeros(horz_n_elems, zelems)
+max_q4 = zeros(horz_n_elems, zelems)
+min_q5 = zeros(horz_n_elems, zelems)
+max_q5 = zeros(horz_n_elems, zelems)
+
+# Initialize pressure and density
+p(z) = p_0 * exp(-z / H)
+ρ_ref(z) = p(z) / R_d / T_0
+
+y0 = map(center_coords) do coord
+    z = coord.z
+    zd = z - z_c
+    λ = coord.long
+    ϕ = coord.lat
+    rd = Vector{Float64}(undef, 2)
+
+    # great circle distances
+    for i in 1:2
+        rd[i] = Geometry.great_circle_distance(coord, centers[i], global_geom)
+    end
+
+    # scaled distance functions
+    d = Vector{Float64}(undef, 2)
+    for i in 1:2
+        d[i] = min(1, (rd[i] / R_t)^2 + (zd / Z_t)^2)
+    end
+
+    q1 = 0.5 * (1 + cos(pi * d[1])) + 0.5 * (1 + cos(pi * d[2]))
+    q2 = 0.9 - 0.8 * q1^2
+    q3 = 0.0
+    if d[1] < 0.5 || d[2] < 0.5
+        q3 = 1.0
+    else
+        q3 = 0.1
+    end
+
+    if (z > z_c) && (ϕ > ϕ_c - rad2deg(1 / 8)) && (ϕ < ϕ_c + rad2deg(1 / 8))
+        q3 = 0.1
+    end
+    q4 = 1 - 3 / 10 * (q1 + q2 + q3)
+    q5 = 1
+
+    ρq1 = ρ_ref(z) * q1
+    ρq2 = ρ_ref(z) * q2
+    ρq3 = ρ_ref(z) * q3
+    ρq4 = ρ_ref(z) * q4
+    ρq5 = ρ_ref(z) * q5
+
+    return (ρ = ρ_ref(z), ρq1 = ρq1, ρq2 = ρq2, ρq3 = ρq3, ρq4 = ρq4, ρq5 = ρq5)
+end
+
+y0 = Fields.FieldVector(
+    ρ = y0.ρ,
+    ρq1 = y0.ρq1,
+    ρq2 = y0.ρq2,
+    ρq3 = y0.ρq3,
+    ρq4 = y0.ρq4,
+    ρq5 = y0.ρq5,
+)
```

Comment:
```suggestion
tracers = map(center_coords) do coord
    z = coord.z
    zd = z - z_c
    λ = coord.long
    ϕ = coord.lat
    rd = Vector{Float64}(undef, 2)

    # great circle distances
    for i in 1:2
        rd[i] = Geometry.great_circle_distance(coord, centers[i], global_geom)
    end

    # scaled distance functions
    d = Vector{Float64}(undef, 2)
    for i in 1:2
        d[i] = min(1, (rd[i] / R_t)^2 + (zd / Z_t)^2)
    end

    q1 = 0.5 * (1 + cos(pi * d[1])) + 0.5 * (1 + cos(pi * d[2]))
    q2 = 0.9 - 0.8 * q1^2
    q3 = 0.0
    if d[1] < 0.5 || d[2] < 0.5
        q3 = 1.0
    else
        q3 = 0.1
    end

    if (z > z_c) && (ϕ > ϕ_c - rad2deg(1 / 8)) && (ϕ < ϕ_c + rad2deg(1 / 8))
        q3 = 0.1
    end
    q4 = 1 - 3 / 10 * (q1 + q2 + q3)
    q5 = 1

    ρq1 = ρ_ref(z) * q1
    ρq2 = ρ_ref(z) * q2
    ρq3 = ρ_ref(z) * q3
    ρq4 = ρ_ref(z) * q4
    ρq5 = ρ_ref(z) * q5

    return (ρq1 = ρq1, ρq2 = ρq2, ρq3 = ρq3, ρq4 = ρq4, ρq5 = ρq5)
end

y0 = Fields.FieldVector(
    ρ = ρ_ref.(center_coords.z),
    tracers = tracers
)
```

Reply:
this will then give you two fields: a scalar field and a tuple-valued field of tracers